modules/test/playwright/tests/export-import-web/pages/ExportImportPage.ts

```diff
@@ -45,6 +45,29 @@ export class ExportImportPage {
 		await this.exportButton.click();
 	}
 
+	async checkItemInNewlyCreatedImportProcess(
+		folderPath: string,
+		itemToCheck: string
+	) {
+		await this.newImportButton.click();
+
+		const fileChooserPromise = this.page.waitForEvent('filechooser');
+
+		await this.fileSelector.click();
+
+		const fileChooser = await fileChooserPromise;
+
+		await fileChooser.setFiles(folderPath);
+
+		await this.continueButton.click();
+
+		await this.page.waitForLoadState('domcontentloaded');
+		await this.page.waitForTimeout(1000);
```

Comment:
is it possible to change the timeout to something else?

Reply:
The number is already a reduced number, but I believe we need that time to upload the file.